tools/build.gradle

```diff
@@ -8,7 +8,7 @@ dependencies {
   api project(':path-queries')
   api project(':utils')
   api project(':openehr-terminology')
-  testImplementation project(':openehr-rm')
+  api project(':openehr-rm')
```

Comment:
If you use reflections (see other comment), this should not be necessary.

Reply:
In fact, this dependency not being here is by design, so we can switch to other RM implementations. So, reflection, or some kind of plugin in the model info lookup is the way to go, instead of this.